lib/customer.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'restaurant'
+
+class Customer
+
+  attr_accessor :order, :total_cost
+
+  def initialize
+    @order = []
+    @total_cost = 0
+  end
+
+  def see_all_dishes(restaurant)
+    restaurant.dishes.each do |item, price|
+      # price = sprintf('%.2f', price)
+      p "#{item} - £#{sprintf('%.2f',price)}"
```

Comment:
Space missing after comma.<br>Do not write to stdout. Use Rails' logger if you want to log.